webglreport.js

```diff
@@ -467,6 +467,13 @@ $(function() {
         });
     }
 
+    if (gl.getError()) {
```

Comment:
Just a question, how sensitive is this line?  Is there some chance that some version of Firefox or some other browser will trip over this and fail the test even when WebGL is working?  I know that Firefox can be pretty verbose with its WebGL warnings, but I don't remember if `gl.getError` is triggered by those warnings or only serious errors.

This might be fine, I just had to ask.